Source/JavaScriptCore/inspector/scripts/codegen/generate_cpp_backend_dispatcher_implementation.py

```diff
@@ -311,7 +311,7 @@ def _generate_dispatcher_implementation_for_command(self, command, domain):
                 parameter_value = parameter_value + '.releaseNonNull()'
             elif CppGenerator.should_dereference_argument(_type, parameter.is_optional):
                 parameter_value = '*' + parameter_value
-            elif CppGenerator.should_move_argument(_type, parameter.is_optional):
+            elif CppGenerator.should_move_argument(_type, parameter.is_optional) or _type.raw_name() == 'string':
```

Comment:
what would happen if we changed `CppGenerator.should_move_argument` to have this instead so that we do it everywhere?